Automattic/wp-calypso
# DO NOT MERGE: Update font sizes to CSS vars

#### Changes proposed in this Pull Request

* Use CSS vars for font sizes and replace existing hard-coded font sizes with variables.
* We may want to use mixins to set line heights and variables rather than setting these independently. I need to investigate more.

#### Testing instructions

* Switch to this PR
* Navigate around WP.com and look for visual bugs. Nothing should have changed from master, this is a straightforward find and replace.
